### PR TITLE
Issue 113

### LINKED ISSUE #113: Main Table Creation - Appointment, Date, Time, WorkSchedule
- [ ] Appointment
- [ ] Date
- [ ] Time
- [ ] WorkSchedule

### PR DESCRIPTION
**Pull request by rko7**
- created appointmnt, time, date, workSchedule schema

### CODE REVIEW

File: models/appointmnt.js [hunk 1]
```diff
@@ -0,0 +1,27 @@
+const mongoose = require('mongoose');
+const Schema = mongoose.Schema;
+
+const AppointmntSchema = new Schema({
```
Comment (eunbeek) [inline models/appointmnt.js]: 'e' missing -> 'Appointment'

File: models/appointmnt.js [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+    },
+    scheduleId: {
+        type: Schema.Types.ObjectId,
+        ref: 'workSchedule',
+        required: true
+    } 
+
+}, { timestamps: true });
+
+const Appointmnt = mongoose.model('appointmnt', AppointmntSchema);
```
Comment (eunbeek) [inline models/appointmnt.js]: collection name is also appointmnt, depending on your desicion, but if you want to use it, appointmnt, please let us know it.

File: models/appointment.js [hunk 3]
```diff
@@ -0,0 +1,27 @@
+const mongoose = require('mongoose');
+const Schema = mongoose.Schema;
+
+const AppointmntSchema = new Schema({
```
Comment (yuanLeeMidori) [inline models/appointment.js]: appointment without e

File: models/appointment.js [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+    },
+    scheduleId: {
+        type: Schema.Types.ObjectId,
+        ref: 'workSchedule',
+        required: true
+    } 
+
+}, { timestamps: true });
+
+const Appointmnt = mongoose.model('appointmnt', AppointmntSchema);
```
Comment (yuanLeeMidori) [inline models/appointment.js]: without e

**Review by yuanLeeMidori — CHANGES_REQUESTED** [review]

**Review by yuanLeeMidori — APPROVED** [review]

**Review by eunbeek — APPROVED** [review]

**Pull request merged**